orbyter-ml-dev/requirements.txt

```diff
@@ -1,43 +1,45 @@
+aquirdturtle-collapsible-headings==3.1.0
 bayesian-optimization==1.2.0
-black==20.8b1
-boto3==1.17.10
+black==21.5b2
+boto3==1.17.92
 click==7.1.2
 coloredlogs==15.0
-dask[complete]==2021.02.0
-dask-ml==1.8.0
-dvc==1.11.16
-flake8==3.8.4
-ipdb==0.13.4
-ipython==7.20.0
+dask[complete]==2021.6.0
+dask-cloudprovider==2021.3.1
+dask-ml==1.9.0
+dvc==2.3.0
+flake8==3.9.2
+ipdb==0.13.9
+ipython==7.24.1
 isort==5.8.0
 jsonlines==2.0.0
 jupyter==1.0.0
 jupyter-contrib-nbextensions==0.5.1
-jupyterlab==3.0.7
-lightgbm==3.1.1
-line-profiler==3.1.0
-matplotlib==3.3.4
-mlflow==1.13.1
-more-itertools==8.7.0
-mypy==0.800
-notebook==6.2.0
-numpy==1.20.1
-pandas==1.2.2
+jupyterlab==3.0.16
+lightgbm==3.2.1
+line-profiler==3.3.0
+matplotlib==3.4.2
+mlflow==1.17.0
+more-itertools==8.8.0
+mypy==0.902
+notebook==6.4.0
+numpy==1.20.3
+pandas==1.2.4
 plotly==4.14.3
 pluggy==0.13.1
 psycopg2==2.8.6
-pyarrow==3.0.0
-pytest==6.2.2
-python-dotenv==0.15.0
-pytictoc==1.5.1
-scikit-learn==0.24.1
-scipy==1.6.0
+pyarrow==4.0.1
+pytest==6.2.4
+python-dotenv==0.17.1
+pytictoc==1.5.2
+scikit-learn==0.24.2
+scipy==1.6.3
 seaborn==0.11.1
-shap==0.38.1
-Sphinx==3.5.1
-SQLAlchemy==1.3.23
+shap==0.39.0
+Sphinx==4.0.2
```

Comment:
Not sure how careful we need to about major version bumps

Reply:
Good question, and this one might be worth looking into, I'll make sure there isn't anything breaking in their changelog